### ISSUE TITLE
github.com/schollz/croc/v6 installed as "v6" rather than "croc"

### ISSUE THREAD

**Issue by zikaeroh** [myitcv/gobin#82]
I used `gobin` to install this binary, but it was installed as `v6` (the version) instead of the actual binary name, `croc`.

```
Installed github.com/schollz/croc/v6@v6.1.1 to /home/jake/dotfiles/gotools/bin/v6
```

**Comment by myitcv** [comment]
Thanks for the report. This is fixed in #83.

**Comment by myitcv** [comment]
Fixed in #83, released in https://github.com/myitcv/gobin/releases/tag/v0.0.12. Thanks again for the report.

**Comment by myitcv** [comment]
You can use `gobin` to upgrade itself:

```
$ gobin -u github.com/myitcv/gobin
Installed github.com/myitcv/gobin@v0.0.12 to /home/myitcv/bin/bingo/gobin
```
